src/data/polynomial/eval.lean

```diff
@@ -223,6 +223,13 @@ begin
   rw [f.map_zero, zero_mul]
 end
 
+lemma eval₂_dvd : p ∣ q → eval₂ f x p ∣ eval₂ f x q
+| ⟨r, h⟩ := ⟨eval₂ f x r, by { rw [h, eval₂_mul] }⟩
```

Comment:
is this worth generalizing as something like the following?
```
-- note: this doesn't actually need `map_one`
lemma monoid_hom.map_dvd {α β} [monoid α] [monoid β] (a b : α) (f : α →* β) : a ∣ b → f a ∣ f b
| ⟨x, h⟩ := ⟨f x, h.symm ▸ f.map_mul _ _⟩
```

Reply:
It turns out we already had this for `ring_hom`s, so I generalized it: #8722

Reply:
Nice find, I didn't think of looking for the ring_hom version first!